server/src/main/java/io/spine/server/commandbus/MatchesCommandBusFilter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.commandbus;
+
+import com.google.common.base.Predicate;
+import io.grpc.stub.StreamObserver;
+import io.spine.base.Command;
+import io.spine.envelope.CommandEnvelope;
+
+import javax.annotation.Nullable;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * A predicate on {@link CommandEnvelope} matching it to a {@link CommandBusFilter}.
+ *
+ * <p>The filter receives the given {@link StreamObserver} for either acknowledging the Command
+ * posting or emitting an error.
+ *
+ * @author Dmytro Dashenkov
+ */
+final class MatchesCommandBusFilter implements Predicate<CommandEnvelope> {
```

Comment:
I wouldn't call it like this. `MatchesCommandBusFilter` looks like a `Filter` implementation, which it isn't.